wp-cli.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+class WPCOM_Compat_Command extends WPCOM_VIP_CLI_Command {
+
+	/**
+	 * Import protected embeds from WordPress.com
+	 *
+	 * # OPTIONS
+	 *
+	 * <file>
+	 * : The CSV file to import
+	 *
+	 * @subcommand import-protected-embeds
+	 */
+	function import_protected_embeds( $args ) {
+		list( $file ) = $args;
+
+		if ( ! file_exists( $file ) ) {
+			WP_CLI::error( 'Specified file does not exist' );
+		}
+
+		
+		$sql = 'CREATE TABLE `protected_embeds` ( ' .
+			'`id` bigint(20) unsigned NOT NULL AUTO_INCREMENT, ' .
+			'`embed_id` varchar(64) NOT NULL, ' .
+			'`src` varchar(255) NOT NULL, ' .
+			'`embed_group_id` varchar(64) NOT NULL, ' .
+			'`html` mediumtext, ' .
+			'`time_added` timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP, ' .
+			'PRIMARY KEY (`id`), ' .
+			'UNIQUE KEY `embed_id` (`embed_id`) ' .
+			') ENGINE=InnoDB AUTO_INCREMENT=0 DEFAULT CHARSET=latin1';
+
+		require_once( ABSPATH . 'wp-admin/includes/upgrade.php' );
+		$q = dbDelta( $sql );
+
+		WP_CLI::line( $q[ 'protected_embeds' ] );
+
+		$fd = fopen( $file, 'r' );
+		if ( ! $fd ) {
+			WP_CLI::error( sprintf( 'Could not open file: %s', $file ) );
+		}
+
+		$header = fgetcsv( $fd );
```

Comment:
Maybe want to verify we have exactly the fields we expect here and probably before we go through the process of creating a new table.

Reply:
💯 -- would definitely be a good idea.